core/pom.xml

```diff
@@ -1,223 +1,211 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-    <modelVersion>4.0.0</modelVersion>
-
-    <artifactId>vitro-languages-core</artifactId>
-    <packaging>pom</packaging>
-
-    <name>Vitro Languages CORE</name>
-    <description>Vitro Languages</description>
-    <url>http://vivoweb.org/</url>
-
-    <licenses>
-        <license>
-            <name>BSD 3-Clause License</name>
-            <url>https://raw.github.com/vivo-project/VIVO/develop/LICENSE</url>
-            <distribution>repo</distribution>
-        </license>
-    </licenses>
-
-    <developers>
-        <developer>
-            <name>Graham Triggs</name>
-            <email>gtriggs@duraspace.org</email>
-            <organization>DuraSpace</organization>
-            <organizationUrl>http://vivoweb.org/</organizationUrl>
-        </developer>
-        <developer>
-            <name>Jim Blake</name>
-            <email>jeb228@cornell.edu</email>
-            <organization>Cornell University Library</organization>
-            <organizationUrl>https://github.com/j2blake</organizationUrl>
-        </developer>
-        <developer>
-            <name>John Fereira</name>
-            <email>jaf30@cornell.edu</email>
-            <organization>Cornell University</organization>
-            <organizationUrl>https://github.com/fereira</organizationUrl>
-        </developer>
-    </developers>
-
-    <!--
-       The SCM repository location is used by Continuum to update against
-       when changes have occurred.  This spawns a new build cycle and releases
-       snapshots into the snapshot repository below.
-    -->
-    <scm>
-        <connection>scm:git:git@github.com:vivo-project/Vitro-languages.git</connection>
-        <developerConnection>scm:git:git@github.com:vivo-project/Vitro-languages.git</developerConnection>
-        <url>git@github.com:vivo-project/Vitro-languages.git</url>
-        <tag>HEAD</tag>
-    </scm>
-
-    <properties>
-        <vitro-version>${project.version}</vitro-version>
-        <vivo-dir>${basedir}</vivo-dir>
-        <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ss</maven.build.timestamp.format>
-        <build.timestamp>${maven.build.timestamp}</build.timestamp>
-        <maven-site-plugin.skip>false</maven-site-plugin.skip>
-        <stagingBase>/</stagingBase>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
-    <modules>
-        <module>webapp</module>
-    </modules>
-
-    <profiles>
-        <profile>
-            <id>release-sign-artifacts</id>
-            <activation>
-                <property>
-                    <name>performRelease</name>
-                    <value>true</value>
-                </property>
-            </activation>
-            <build>
-                <plugins>
-                    <!-- plugin>
```

Comment:
Is this intentional? Do you really prefer to have XML elements compacted?

Reply:
Sorry but i do not see exactly what is the problem

Reply:
The potential problem is that this `plugin` has been changed to being collapsed: https://github.com/vivo-project/Vitro-languages/pull/12/files#diff-357e4854869b2e21c38b1b437f11095aR77-R81

That does not seem like an improvement.

The same is true for the other spots where I made the same comment.